runtime/karura/src/lib.rs

```diff
@@ -1439,7 +1447,7 @@ impl xcm_executor::Config for XcmConfig {
 	type Barrier = Barrier;
 	type Weigher = FixedWeightBounds<UnitWeightCost, Call>;
 	// Only receiving KSM is handled, and all fees must be paid in KSM.
-	type Trader = FixedRateOfConcreteFungible<KsmPerSecond, ToTreasury>;
+	type Trader = FixedRateOfConcreteFungible<BifrostNativeTokenLocation, ToTreasury>;
```

Comment:
This breaks KSM transfer between Karura and Kusama. Should use multiple traders instead. Blocked by https://github.com/paritytech/polkadot/pull/3601